matching-notification-api/src/main/java/org/phenotips/matchingnotification/match/internal/DefaultPatientInMatch.java

```diff
@@ -401,6 +411,20 @@ private void readDetails(Patient patient)
         }
     }
 
+    private Set<String> getMatchedExomeGenes(Set<String> matchedGenes)
+    {
+        if (this.hasExomeData()) {
+            // we assume that matched exome genes are all that are not candidate/solved
```

Comment:
Assumptions are bad. Can't this info be made explicit? By using a specific status?